activesupport/lib/active_support/core_ext/module/attribute_accessors.rb

```diff
@@ -8,6 +8,8 @@ class Module
   # Defines a class attribute and creates a class and instance reader methods.
   # The underlying class variable is set to +nil+, if it is not previously
   # defined.
+  # Note: Both methods will be public even if this method is called from
+  # inside a private or protected block.
```

Comment:
> private and protected are just methods, not some sort of special declaration.

That is nice for general education in the inner workings of ruby, but not very helpful in answering the original question, which is how the sentence could be better phrased.